airbyte-integrations/connectors/source-plaid/integration_tests/catalog.json

```
@@ -105,6 +106,12 @@
             "check_number": { "type": ["string", "null"] },
             "datetime": { "type": ["string", "null"] },
             "merchant_name": { "type": ["string", "null"] },
+            "name": { "type": ["string"] },
+            "website": { "type": ["string", "null"] },
+            "merchant_entity_id": { "type": ["string", "null"] },
+            "logo_url": { "type": ["string", "null"] },
+            "personal_finance_category_icon_url": { "type": ["string"]},
+            "transaction_type": { "type": ["string"]},
```

Same with these two: no array needed.

```suggestion
            "personal_finance_category_icon_url": { "type": "string" },
            "transaction_type": { "type": "string" },
```